catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -76,4 +84,35 @@ protected void printErrorMessage(String message) {
         printColor(ERROR_COLOR, message);
     }
 
+    protected void printInfoMessage(String message) {
+        printColor(INFO_COLOR, message);
+    }
+
+    private SolrClient getCloudSolrClient() {
+        String zkHosts = System.getProperty(ZOOKEEPER_HOSTS_PROP);
+        LOGGER.debug("Zookeeper hosts: {}", zkHosts);
+
+        if (zkHosts != null) {
+            SolrClient client = new CloudSolrClient.Builder().withZkHost(zkHosts)
+                    .build();
+            LOGGER.debug("Created solr client: {}", client);
+            return client;
+        } else {
+            printErrorMessage(String.format(
+                    String.format("Could not determine Zookeeper Hosts. Please verify that the system property %s is configured in %s.",
+                    ZOOKEEPER_HOSTS_PROP, Paths.get(System.getProperty("ddf.home"), "etc", "system.properties"))));
+            return null;
+        }
+    }
+
+    void shutdown(SolrClient client) throws IOException {
+        if(client != null) {
+            LOGGER.debug("Closing solr client");
+            client.close();
+        }
+    }
+
+    SolrClient getSolrClient() {
+        return getCloudSolrClient();
```

Comment:
❓ Could we reuse the `SolrClientFactory` service to get the proper type of `SolrClient` and use it to perform the backup? Feels to me that all this client building and backup logic could be abstracted behind those classes instead of implemented in the command itself. Thoughts?